main.go

```diff
@@ -70,28 +70,47 @@ type oidcConfig struct {
 	password     string
 }
 
+func LookupEnvOrString(key string, defaultVal string) string {
```

Comment:
Can we please make all of these lowercase, ie non-exported functions? That's the standard for Go and for the Observatorium project if they are only used internally

Reply:
Changes have been made and pushed.